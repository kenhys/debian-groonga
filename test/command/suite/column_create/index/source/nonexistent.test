table_create Memos TABLE_NO_KEY
column_create Memos content COLUMN_SCALAR Text

table_create Terms TABLE_PAT_KEY ShortText \
  --default_tokenizer TokenBigram \
  --normalizer NormalizerAuto
column_create Terms memos_index COLUMN_INDEX|WITH_POSITION Memos nonexistent

dump
