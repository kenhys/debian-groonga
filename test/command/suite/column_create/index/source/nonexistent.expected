table_create Memos TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Memos content COLUMN_SCALAR Text
[[0,0.0,0.0],true]
table_create Terms TABLE_PAT_KEY ShortText   --default_tokenizer TokenBigram   --normalizer NormalizerAuto
[[0,0.0,0.0],true]
column_create Terms memos_index COLUMN_INDEX|WITH_POSITION Memos nonexistent
[[[-22,0.0,0.0],"[column][create] nonexistent source: <nonexistent>"],false]
#|e| [column][create] nonexistent source: <nonexistent>
dump
table_create Memos TABLE_NO_KEY
column_create Memos content COLUMN_SCALAR Text

table_create Terms TABLE_PAT_KEY ShortText --default_tokenizer TokenBigram --normalizer NormalizerAuto
